web/packages/plugins/analysis-platform/src/modules/Analysis/components/CreateJobButton.tsx

```diff
@@ -23,6 +23,7 @@ export const CreateJobButton = (props: {
 }) => {
   const { analysis, addJob, jobs, dataSourceId } = props
   const [loading, setLoading] = useState<boolean>(false)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore